### DISCUSSION TITLE
Copy styles - Firefox

### DISCUSSION THREAD

**Discussion by jives00** [stylus/stylus#2706] [Q&A] [answered]
My PC crashed and I had to reinstall Windows.  Fortunately I have a full backup so I grabbed my Firefox profile and restored it and got back all of my Firefox settings, extensions, etc.... except all my Stylus profiles are missing.

Is there a way I can restore them?  I did not have an export/sync to cloud turned on.  But I do have a full backup of my PC from before the crash available and can grab any file from there.

**Comment by iChenLei** [comment, marked as answer]
@jives00 https://github.com/openstyles/stylus
